Neos.Media/Classes/Domain/Model/ThumbnailConfiguration.php

```diff
@@ -163,6 +172,14 @@ public function getQuality()
         return $this->quality;
     }
 
+    /**
+     * @return string|null
+     */
+    public function getFormat()
```

Comment:
declare return type, please… Flow master requires PHP 7.1 already, so we can use nullable return types. 🎉

Reply:
I did not add those because no other method on there used return types. Added them now